src/components/common/DelayedRender.ts

```diff
@@ -0,0 +1,17 @@
+import { Fragment, PropsWithChildren, useEffect, useState } from 'react'
+import { h } from 'react-hyperscript-helpers'
+
+
+type DelayedRenderProps = PropsWithChildren<{
+  delay?: number
+}>
+
+export const DelayedRender = ({ children = null, delay = 1000 }: DelayedRenderProps) => {
+  const [shouldRender, setShouldRender] = useState(false)
+  useEffect(() => {
+    const timeout = setTimeout(() => setShouldRender(true), delay)
+    return () => clearTimeout(timeout)
+  }, [delay])
+
+  return shouldRender ? h(Fragment, [children]) : null
```

Comment:
Had to change this line from `shouldRender && children` to `shouldRender ? h(Fragment, [children]) : null` to make the types for react-hyperscript-helpers happy.

Reply:
I wonder if a maybeRender(condition: boolean, ...) helper would be good to automatically return the null on false?

Reply:
I'm not sure that's necessary. React is fine with returning false here, it's just our react-hyperscript-helpers types that are unhappy with it.